velox/exec/tests/AggregationFuzzer.cpp

```diff
@@ -419,23 +419,34 @@ void persistReproInfo(
     return;
   }
 
-  // Save input vector.
-  auto inputPathOpt = common::generateTempFilePath(basePath.c_str(), "vector");
-  if (!inputPathOpt.has_value()) {
-    inputPath = "Failed to create file for saving input vector.";
-  } else {
-    inputPath = inputPathOpt.value();
-    try {
-      // TODO Save all input vectors.
-      saveVectorToFile(input.front().get(), inputPath.c_str());
-    } catch (std::exception& e) {
-      inputPath = e.what();
+  // Create a new directory
+  auto dirPath =
+      common::generateTempFolderPath(basePath.c_str(), "expressionVerifier");
+  if (!dirPath.has_value()) {
+    LOG(INFO) << "Failed to create directory for persisting repro info.";
+    return;
+  }
+
+  for (auto i = 0; i < input.size(); ++i) {
+    // Save input vector.
+    auto inputPathOpt =
```

Comment:
Wont it also matter the order in which these inputs are loaded ? If so we should probably add some suffix which tells us order in which to load these inputs ?

Reply:
The order of inputs shouldn't matter for aggregate functions that are not order sensitive. For those that are, we treat them specially by either sorting the inputs before aggregation or skipping the result verification.